docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc

```diff
@@ -20,36 +20,49 @@ There are four steps to set up the Time Series DB for your system:
 . Obtain an activation key for the Cloud Services Connector.
 . Enable cloud services connectivity.
 
-=== Update OpenNMS Cloud Services Connector files
+=== Update Cloud Services Connector files
 
-The OpenNMS Cloud Services Connector is an extension that lets a {page-component-title} instance send time series data to the OpenNMS-hosted cloud service.
+The OpenNMS Cloud Services Connector is an extension that lets a {page-component-title} instance send time series data to the {page-component-title}-hosted cloud service.
 The packages for this extension are installed by default in Horizon 31 and later, but do need to be configured and activated.
 You can use `yum` or `apt` to make sure the `opennms-cloud-plugin` package is up to date with the most current version of the plugin.
 
 To ensure that the feature is activated on service restarts, add the feature to a file in `featuresBoot.d`:
 
 [source, console]
-----
 echo "opennms-plugin-cloud-core wait-for-kar=opennms-plugin-cloud" | sudo tee ${OPENNMS_HOME}/etc/featuresBoot.d/plugin-cloud.boot
-----
 
-=== Configure OpenNMS Cloud Services settings
+=== Configure Cloud Services
 
 You will need to add settings to tell your {page-component-title} instance to load the plugin on service start and to direct storage to the cloud service.
-
 If you are running Horizon 31 or later, copy the example `timeseries.properties` file to your working directory:
 
 [source, console]
-----
 cp ${OPENNMS_HOME}/etc/examples/opennms.properties.d/hosted_tsdb.timeseries.properties ${OPENNMS_HOME}/etc/opennms.properties.d/timeseries.properties
+
+The following tags can be configured in `$\{OPENNMS_HOME}/etc/opennms.properties.d/timeseries.properties`:
+
+* Asset
+* Interface
+* Node
+* Service
+* Resources
+* Custom
+
+Below are examples of tag definitions:
+
+[source, properties]
+----
+org.opennms.timeseries.tin.metatags.tag.node=${node:label}
```

Comment:
should we use the tags listed in the bulleted list as part of the example tag definitions?

Reply:
I think having these should communicate the concept, but I can update this to match the list if it would make more sense. :)

Reply:
No, that's ok. Thanks!